package.json

```diff
@@ -1,11 +1,31 @@
 {
   "name": "sous-project",
-  "license": "MIT",
+  "version": "0.0.1",
+  "description": "A starter kit for a Drupal project",
   "scripts": {
-    "confim": "./scripts/sous/confim.sh",
-    "confex": "./scripts/sous/confex.sh",
-    "import-data": "./scripts/sous/import-data.sh",
-    "rebuild": "./scripts/sous/rebuild.sh",
-    "local-data-bak": "./scripts/sous/local-data-bak.sh"
+    "publish": "semantic-release --tag-format '${version}'",
+    "publish-test": "semantic-release --tag-format '${version}' --dry-run --debug",
+    "postinstall": "patch-package"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:fourkitchens/sous-drupal-project.git"
+  },
+  "keywords": [
+    "sous"
+  ],
+  "author": "Four Kitchens",
+  "license": "GPL-2.0-or-later",
```

Comment:
GLPV2 and later versions are not compatible iirc, I wonder if we should just use what Drupal standardizes upon?

Reply:
@patrickocoffeyo I thought GPL is what Drupal uses (https://www.drupal.org/about/licensing#drupal-license), do you have a licence that you like to use?

Reply:
Ah interesting, Drupal allows projects to select which version of GLP they use, and `GPL-2.0-or-later` is a valid SPDX license, so this is :100:!